php/commands/site.php

```diff
@@ -985,33 +987,67 @@ public function import($args, $assoc_args) {
    * [--site=<site>]
    * : Site to use
    *
-   * [--change-to-org=<org>]
-   * : Change the instrument to an Org by setting the id. ( must be admin )
+   * [--instrument=<UUID>]
+   * : Change the instrument by setting the ID
    *
    * ## EXAMPLES
    *
    *  terminus site instrument --site=sitename
    */
   public function instrument($args, $assoc_args) {
-    $site = SiteFactory::instance( Input::site( $assoc_args ) );
-    $org = Input::optional('change-to-org', $assoc_args);
-    $data = $site->instrument($org);
-    // @TODO we need a "workflow" class to handle these exceptions and whatnot
-    if ($org) {
-      if ( 'failed' == $data->result || 'aborted' == $data->result ) {
-        if (isset($data->final_task) AND !empty($data->final_task->messages)) {
-          foreach( (array) $data->final_task->messages as $date => $message) {
-            \Terminus::error('%s', $message->message);
+    $user = new User();
+    $instruments = $user->instruments()->all();
+    foreach($instruments as $instrument) {
+      $data[$instrument->get('id')] = $instrument->get('label');
```

Comment:
yeah, I'm not sure how we can put PHP in FREAKING_STRICT mode and have it exit on everything. It's too tankish for my taste.